app/invocation/invocation.tsx

```diff
@@ -113,6 +112,10 @@ export default class InvocationComponent extends React.Component<Props, State> {
     }, 3000);
   }
 
+  getActiveTab(): InvocationTabHash {
```

Comment:
Doesn't the tab component need to know the active tab too - can we just move all of this logic there?

Reply:
Good call -- done